extensions/src/AWSSDK.Extensions.NETCore.Setup/AWSSDK.Extensions.NETCore.Setup.csproj

```diff
@@ -13,6 +13,10 @@
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
   </PropertyGroup>
+	
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
```

Comment:
Should we set `<WarningsAsErrors>IL2026</WarningsAsErrors>` anywhere we're adding this? 

We're annotating everything today, but as long as these are warnings I think it'd be easy for somebody to add new functionality that is not compatible and not think about this.

Reply:
Yes once all of these warnings have been addressed. I still have a few more in Core that need to be addressed. The whole dynamic loading of assemblies like STS.